Godeps/Godeps.json

```diff
@@ -128,13 +128,18 @@
 		},
```

Comment:
@unclejack I don't think what you're asking is always possible. Many times godep update will require corresponding changes to just make things compile.